src/main/java/uk/gov/justice/hmpps/prison/service/SchedulesService.java

```diff
@@ -72,7 +72,7 @@ public SchedulesService(final LocationService locationService,
         this.maxBatchSize = maxBatchSize;
     }
 
-    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
The endpoint that calls this method is only ever called by prisonstaffhub which does not have SYSTEM_USER